crm_rma_claim_make_claim/model/crm_claim_rma.py

```diff
@@ -0,0 +1,145 @@
+                for claim_line_id in partner_claim_line_ids.get(partner_id):
+
+                    # Search if have claim line have a child
+                    claim_lines_exists = claim_line_obj.search(
+                        [('claim_line_id', '=', claim_line_id)],)
+                    claim_lines_exists = [cla.id for cla in claim_lines_exists]
+
+                    # If claim_line_id does not have child
+                    if not claim_lines_exists:
+                        # The claim line of supplier claim is created
+                        claim_line_id = claim_line_obj.browse(claim_line_id)
+
+                        # invl = \
+                        #     invline_obj.search([('invoice_id',
+                        #                         '=',
+                        #                          claim_line_id.
+                        #                          supplier_invoice_id.id),
+                        #                        ('product_id',
+                        #                         '=',
+                        #                         claim_line_id.product_id.id
+                        #                         )])
+                        claim_line_new = \
+                            claim_line_id.copy({
+                                'claim_id': False,
+                                'number': '/',
+                                'claim_type':
+                                self.env.ref('crm_claim_rma_config.'
+                                             'crm_claim_type'
+                                             '_supplier').id,
+                                'claim_line_id': claim_line_id.id,
+                                # 'state': 'draft',
+                                # 'invoice_id': claim_line_id.
+                                # supplier_invoice_id.id,
+                                # 'supplier_invoice_id': inv_obj,
+                                # 'invoice_line_id': invl[0].id,
+                            })
+
+                        # is added for display in the view
+                        good_lines.append(claim_line_new.id)
+                    else:
+                        # Anyway. If claim line have child,
+                        # is added for display in the view
+                        good_lines += claim_lines_exists
+
+            if not good_lines:
+                raise except_orm(
+                    _('Error'),
+                    _('The Supplier Claim Was Not Created.'))
+            good_lines = list(set(good_lines))
+            mod_obj = self.env['ir.model.data']
+            act_obj = self.env['ir.actions.act_window']
+
+            result = mod_obj.get_object_reference('crm_claim_rma',
+                                                  'act_crm_case_claim_lines')
+            act_id = result and result[1] or False
+            act_rec = act_obj.browse(act_id)
+            result = act_rec.read()
```

Comment:
these 7 lines can be replaced by `self.env.ref('crm_claim_rma.act_crm_case_claim_lines')` [`.ref` doc](https://www.odoo.com/documentation/8.0/reference/orm.html#common-orm-methods)